### ISSUE TITLE
NotSupportedError: Failed to load because no supported source was found.

### ISSUE THREAD

**Issue by joowols** [FreeTubeApp/FreeTube#5463]
### Guidelines

- [X] I have encountered this bug in the [latest release of FreeTube](https://github.com/FreeTubeApp/FreeTube/releases).
- [X] I have encountered this bug in the [official downloads of FreeTube](https://github.com/FreeTubeApp/FreeTube#official-downloads).
- [X] I have searched the issue tracker for [open](https://github.com/FreeTubeApp/FreeTube/issues?q=is%3Aopen+is%3Aissue) and [closed](https://github.com/FreeTubeApp/FreeTube/issues?q=is%3Aissue+is%3Aclosed) issues that are similar to the bug report I want to file, without success.
- [X] I have searched the [documentation](https://docs.freetubeapp.io/) for information that matches the description of the bug I want to file, without success.
- [X] This issue contains only one bug.

### Describe the bug

subscription page. click on video. video comes up.
click on video, doesnt play, error comes up-'NotSupportedError: Failed to load because no supported source was found.

### Expected Behavior

I expected the video to start playing

### Issue Labels

accessibility issue, usability issue

### FreeTube Version

0.21.1

### Operating System Version

windows 10 home 22H2

### Installation Method

.exe

### Primary API used

Local API

### Last Known Working FreeTube Version (If Any)

0.20

### Nightly Build

- [X] I have encountered this bug in the latest [nightly build](https://docs.freetubeapp.io/development/nightly-builds).

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
Duplicate of https://github.com/FreeTubeApp/FreeTube/issues/5456

[Temporary workaround](https://github.com/FreeTubeApp/FreeTube/issues/5456#issuecomment-2248897869)
